src/bundle/Resources/public/js/CKEditor/custom-tags/inline-custom-tag/inline-custom-tag-editing.js

```diff
@@ -0,0 +1,146 @@
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import { toWidget } from '@ckeditor/ckeditor5-widget/src/utils';
```

Comment:
what would you say on moving this line below? I personally prefer having import utils below imports of classes